js/spectrogram/Spectrogram.js

```diff
@@ -187,43 +174,54 @@ class Spectrogram { // eslint-disable-line no-unused-vars
     this.getFundamental(this.fft.data);
   }
 
-  // get the scale of the canvas.
-  // That is, how much do I need to multiply by to fill the screen from the fft.data
+  /**
+   * Gets the scale for the canvas.
+   *
+   * This method calculates the scaling factor for the spectrogram and the scale/ruler, i.e, what
+   * the fft.data values need to be multiplied by to fill the screen.
+   *
+   * @todo Check if this.scaleX is necessary.
```

Comment:
It's unused for now because the 1D FFT is not yet implemented. That said, the FFT would probably have its own scale update since they have their own windows